template/before.sh.erb

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+export SING_OVERLAYS=""
+
+## create custom desktop folder
+export DESKTOP_FOLDER="${PWD}/desktop"
+export XDG_DESKTOP_DIR=$DESKTOP_FOLDER
+mkdir -p $DESKTOP_FOLDER
+echo "TIMING: $(date -Iseconds) - set up desktop folder"
+
+## create the configuration folder
+export XDG_CONFIG_HOME=$PWD/config
+mkdir -p $XDG_CONFIG_HOME
+export XDG_CONFIG_DIRS=$XDG_CONFIG_HOME
+echo "TIMING: $(date -Iseconds) - set up config folder"
+
+## we set a separate desktop folder, but unique Documents , etc etc 
+cat > $XDG_CONFIG_DIRS/user-dirs.dirs  <<EOF
+XDG_DESKTOP_DIR="$DESKTOP_FOLDER"
+XDG_DOCUMENTS_DIR="$HOME/Documents"
+XDG_DOWNLOAD_DIR="$HOME/Downloads"
+XDG_MUSIC_DIR="$HOME/Music"
+XDG_PICTURES_DIR="$HOME/Pictures"
+XDG_PUBLICSHARE_DIR="$HOME/Public"
+XDG_TEMPLATES_DIR="$HOME/Templates"
+XDG_VIDEOS_DIR="$HOME/Videos"
+EOF
+echo "TIMING: $(date -Iseconds) - set up user dirs folder"
+
+# Create a desktop icon for Gazebo
+cat > $DESKTOP_FOLDER/Gazebo.desktop <<EOF
```

Comment:
Is the Gazebo desktop icon supposed to work for pyraf? There's also a VSCodium and Matlab desktop icon.

Reply:
Codium wouldn't be bad to have, but I don't see it getting installed in the container definition. Gazebo and Matlab aren't getting installed either, as far as I can see.

Reply:
Must have missed this. Will remove as we dont have a use for Gazebo, matlab, or VScodium